src/app/dim-ui/Sheet.tsx

```diff
@@ -238,66 +239,70 @@ export default function Sheet({
     };
   }, []);
 
+  const sheetBody = (
```

Comment:
I did this just so in the future, adding more providers to the root of the sheet won't re-indent everything